src/test_helpers/integration.cljs

```diff
@@ -6,11 +6,24 @@
     legacy.status-im.subs.root
     [native-module.core :as native-module]
     [re-frame.core :as rf]
+    status-im.contexts.onboarding.events
     status-im.events
     status-im.navigation.core
     status-im.subs.root
     [taoensso.timbre :as log]
-    [tests.integration-test.constants :as constants]))
+    [tests.integration-test.constants :as constants]
+    [utils.security.core :as security]
+    [utils.transforms :as transforms]))
+
+(defn validate-mnemonic
+  [mnemonic on-success on-error]
+  (native-module/validate-mnemonic
```

Comment:
I added this over the reframe event that we have because the reframe event is coupled with an options menu opening up. 
Additionally I thought it best not to add re-frame events just for the sake of testing.